oak_dice/src/cert.rs

```diff
@@ -47,6 +47,9 @@ pub const MEMORY_MAP_MEASUREMENT_ID: i64 = -4670559;
 /// ID for the CWT private claim ID corresponding to the hash of the commands for building the ACPI
 /// tables.
 pub const ACPI_MEASUREMENT_ID: i64 = -4670560;
+/// ID for the CWT private claim label corresponding to the hash of the binary for the layer in the
+/// case where a single binary is measured for a layer..
+pub const CODE_DIGEST_ID: i64 = -4670545;
```

Comment:
Is this the Layer 1 code digest ID? If not, then where is that ID defined?
If yes, as per [go/tee-dice-attestation](http://goto.google.com/tee-dice-attestation) it seems it should be -4670567.

Reply:
Good catch. This is the ID used for generic code measurements in the Open DICE profile, but we agreed to use unique IDs for the different layers. I will update in a follow-up PR.